### DISCUSSION TITLE
Source code for Icons/Themes/Expo/Blog sections of getbootstrap.com?

### DISCUSSION THREAD

**Discussion by techboyg5** [twbs/bootstrap#31555] [General] [answered]
Hi there! I noticed that this GitHub repository only includes code for the Home, Documentation and Examples sections of [getbootstrap.com](https://getbootstrap.com/).

Where is the source code for the Icons, Themes, Expo and Blog sections?

**Comment by patrickhlauke** [comment]
in the various other repos https://github.com/twbs

**Reply by techboyg5** [reply]
You partially answered my question. I can find the Icons section at twbs/icons, the Expo section at twbs/expo, and the Blog section at twbs/blog.

But where is the Themes section? I can't find it at twbs/themes.

**Comment by patrickhlauke** [comment, marked as answer]
themes are a commercial product. not going to just make their source readily available...

**Reply by techboyg5** [reply]
> 
> 
> themes are a commercial product. not going to just make their source readily available...

I'm not talking about the themes themselves. I'm talking about [themes.getbootstrap.com](https://themes.getbootstrap.com/).

**Reply by Deckluhm** [reply]
https://themes.getbootstrap.com is a WordPress so source is probably not available anywhere.

**Reply by mdo** [reply]
Patrick's response applies to both the Themes project site and the themes we sell on the marketplace.

**Reply by techboyg5** [reply]
OK, thanks! But I think that making the templates of the themes section open source is a good idea.
